### ISSUE TITLE
[API] Add product images in product serialization

### ISSUE THREAD

**Issue by jmaupetit** [thelia/thelia#1813]
I understand that accessing product images via a dedicated endpoint is useful, but what about embedding product image objects in the serialized product itself to avoid multiple requests when iterating over a product list?

**Comment by bibich** [comment]
Yes, it would be great to have a dedicated endpoint to get all information of a product (images, documents, product sale elements, ...) that also collects specific data added by modules.
